lib/main.dart

```diff
@@ -124,53 +129,57 @@ class _Header extends StatelessWidget with GetItMixin {
               Padding(
                 padding:
                     const EdgeInsets.symmetric(horizontal: 24, vertical: 24),
-                child: Row(
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: [
-                    InkWell(
-                      onTap: () async {
-                        await launch('https://www.devshelpdevs.org');
-                      },
-                      child: SvgPicture.asset(
-                        'assets/images/devshelpdevs-logo.svg',
-                        height: 100,
-                      ),
-                    ),
-                    Padding(
-                      padding: const EdgeInsets.only(top: 0),
-                      child: TextButton(
-                        onPressed: () async {
-                          await launch('https://paypal.me/pools/c/8xPwkVP3th');
+                child: SingleChildScrollView(
```

Comment:
Why a scrollview in the Header?

Reply:
Good day sir,

The header line overflows the width of the screen and for now, I just decided to give it a scroll view to make it scrollable.

However, in my next commit, I plan to make the header adapt in size either through the use of layout breakpoints or specifying the width per screen width type

Reply:
Ah. I missed that it's a horizontal one.
Probably we have to use a totally different header widget for mobile

Reply:
That's a more preferable option sir